src/components/no-longer-live/no-longer-live.scss

```diff
@@ -3,7 +3,7 @@
   position: absolute;
   top: 0;
   right: 0;
-  z-index: 5; // overlap spinner and replay button
+  z-index: 6; // overlap spinner and replay button
```

Comment:
@semarche-kaltura not something major, but i would consider use a variable since we're using it in more than 1 place.

Reply:
good catch, I'll update PR